File: Programming Language(ver.2)/PHP/305.php
<?php
    function compute($a, $operator, $b){
        $result = 0;
        if($operator == 1){
            $result = $a + $b;
        }else{
            $result = $a * $b;
        }
        return $result;
    }
    $a = intval(fgets(STDIN));
    $operator = intval(fgets(STDIN));
    $b = intval(fgets(STDIN));
    $result = compute($a, $operator, $b);
    printf("%d\n", $result)
?>

File: Programming Language(ver.2)/Go/305.go
package main
import "fmt"

func compute(array [3]int) int {
	if array[1] == 1{
		return array[0] + array[2]
	}else{
		return array[0] * array[2]
	}
}

func main(){
	var array [3]int
	for i := 0; i < 3; i++{
		fmt.Scanln(&array[i])
	}
	fmt.Println(compute(array))
}
